### ISSUE TITLE
Sidenotes support in tufte

### ISSUE THREAD

**Issue by sivakon** [tlienart/FranklinTemplates.jl#35]

**Comment by tlienart** [comment]
Hello, yes that could be a good addition. Would you be willing to experiment with it? If you can come up with a simple proof of concept, I should be able to include it reasonably quickly.

I think the key obstacle is to get the  page geometry right, there  should be a spot for the main text-content and a spot for the margin where notes get written. This can be copied from however they do  it in the original tufte template though.

**Comment by sivakon** [comment]
I found this. This uses pp preprocesser to generate HTML from pandoc. Basically create divtags with id for each side note.

https://github.com/duzyn/tufte-markdown/blob/master/pp-macros/html/tufte/sidenote.html.pp

**Comment by tlienart** [comment]
yes adding the sidenote itself as an element is trivial (what is done in that gist), the part that isn't is to get it in the right location. The useful bit is to look at how they do it in the original Tufte and just copy that

**Comment by trentfridey** [comment]
Stumbled into this issue when trying out Franklin.jl with tufte, I noticed that the counter `sidenote-counter` declared [in tufte.css](https://github.com/tlienart/FranklinTemplates.jl/blob/e1b28a1b127650cbe3cbb2dfcbf93f77341e3116/src/templates/tufte/_css/tufte.css#L51) was causing the equation number counter to fail, resulting in every equation having number (1)

I don't know where the proper solution lies for this, but I was able to get the equation number counter working again by removing `counter-reset: sidenote-counter` from the `body` tag CSS.

**Comment by tlienart** [comment]
Thanks a lot @trentfridey !
